build.gradle

```diff
@@ -50,15 +50,15 @@ dependencies {
 
     implementation 'com.fasterxml.jackson.datatype:jackson-datatype-jsr310'
 
-    // apache commons.lang3
+    // commons-io
+    implementation 'commons-io:commons-io:2.13.0'
```

Comment:
Why do we need it now? Was it included in Maven-Shared-Utils?

Reply:
Exactly, please see commit 3fe0059 which contains the explanation